mobile/src/main/java/ch/epfl/sdp/mobile/application/chess/notation/FenNotation.kt

```diff
@@ -0,0 +1,100 @@
+package ch.epfl.sdp.mobile.application.chess.notation
+
+import ch.epfl.sdp.mobile.application.chess.engine.Board
+import ch.epfl.sdp.mobile.application.chess.engine.Color
+import ch.epfl.sdp.mobile.application.chess.engine.Piece
+import ch.epfl.sdp.mobile.application.chess.engine.Position
+import ch.epfl.sdp.mobile.application.chess.notation.FenNotationCombinators.activeColor
+import ch.epfl.sdp.mobile.application.chess.notation.FenNotationCombinators.board
+import ch.epfl.sdp.mobile.application.chess.notation.FenNotationCombinators.castlingRights
+import ch.epfl.sdp.mobile.application.chess.notation.FenNotationCombinators.enPassant
+import ch.epfl.sdp.mobile.application.chess.notation.GenericNotationCombinators.integer
+import ch.epfl.sdp.mobile.application.chess.notation.GenericNotationCombinators.spaces
+import ch.epfl.sdp.mobile.application.chess.parser.Combinators.flatMap
+import ch.epfl.sdp.mobile.application.chess.parser.Combinators.map
+
+/**
+ * An object which contains some utilities to transform FEN (Forsyth–Edwards Notation) notation into
+ * games
+ */
+object FenNotation {
+
+  /**
+   * Represents castling rights for a board state
+   * @param kingSideWhite Indicates if white can castle king side
+   * @param queenSideWhite Indicates if white can castle queen side
+   * @param kingSideBlack Indicates if black can castle king side
+   * @param queenSideBlack Indicates if black can castle queen side
+   */
+  data class CastlingRights(
+      val kingSideWhite: Boolean,
+      val queenSideWhite: Boolean,
+      val kingSideBlack: Boolean,
+      val queenSideBlack: Boolean,
+  )
+
+  // TODO: Need to accommodate for whose turn to play, castling rights etc...
```

Comment:
What is this comment for exactly ?

Reply:
It's an artifact of my thought process, for when I thought I would return a `Game` instead of a `Board`. It's gotta go.

Reply:
Addressed in [67a6ba8](https://github.com/epfl-SDP/android/pull/283/commits/67a6ba8e69ab70d1449265ec841c3786d91bddf5).